### ISSUE TITLE
OS X support

### ISSUE THREAD

**Issue by jsilverMDX** [infinitered/rmq#283]
Hi guys I was reading the site and saw this:

"RMQ only works in iOS, not OS X"

Is there any reason for this? I don't need it currently but we use RMQ in our iOS app and love it, so it would be great to use in OS X too if that need arises.

Is it a technical barrier? Is there anyway I can contribute here? If it's simply impossible I understand.

Figured it was worth an ask.

Thanks.

@jsilvermdx

**Comment by twerth** [comment]
Yeah, technically it's NSView vs UIView, etc. It'd have to be ported.

**Comment by jsilverMDX** [comment]
Interesting. Is there a good starting point for this? I'd like to maybe use RMQ in my OS X apps. If directed the right way and I have time, I'd have no problem contributing the patch. But currently I know nothing about the RMQ code base (not even how it works.) If it's not too much code I could just read it tho. :D ?

**Comment by kemiller** [comment]
Probably best to just dive in.  It probably would not be that hard, but you'd need to write stylers and whatnot for the NS* hierarchy, as well as mixins.  The basic mechanics should be pretty similar if not identical.

**Comment by jsilverMDX** [comment]
Cool, thanks. I will give it a shot at soonest convenience. This would be big for RMQ, imo. :D

**Comment by GantMan** [comment]
Hey jsilver.  We're here to help. Have you joined the motioneers chat room?  We can discuss more there :)

**Comment by jsilverMDX** [comment]
I'm there. Thanks Gant :)
